ODIN_II/SRC/odin_ii.cpp

```diff
@@ -607,6 +628,14 @@ void get_options(int argc, char** argv) {
     if (global_args.permissive.value()) {
         warning_message(PARSE_ARGS, unknown_location, "%s", "Permissive flag is ON. Undefined behaviour may occur\n");
     }
+
+    if (global_args.mix_multipliers) {
+        int bit_value = (1 << mix_hard_blocks::MULTIPLIERS);
```

Comment:
I'm not sure what this does?

Reply:
@jeanlego This is for packing enabled optimizations into one config variable that would allow not defining tons of new control config variables to odin_config

Reply:
I like the idea, but I think it affects readability. 
I would just get rid of it since you already have to store some variable to tweak it anywho

Reply:
Done!